src/event/ngx_event.c

```diff
@@ -603,7 +603,11 @@ ngx_event_process_init(ngx_cycle_t *cycle)
     ccf = (ngx_core_conf_t *) ngx_get_conf(cycle->conf_ctx, ngx_core_module);
     ecf = ngx_event_get_conf(cycle->conf_ctx, ngx_event_core_module);
 
-    if (ccf->master && ccf->worker_processes > 1 && ecf->accept_mutex) {
+    if (ccf->master && ccf->worker_processes > 1 && ecf->accept_mutex
+#if (NGX_HAVE_REUSEPORT)
+        && !ecf->reuse_port
+#endif
+       ) {
```

Comment:
@gwtony could you update code style? Thx.
The '{' should be located in a separate line and aligned with `if`.  For example:
```
    if (ccf->master && ccf->worker_processes > 1 && ecf->accept_mutex
#if (NGX_HAVE_REUSEPORT)
        && !ecf->reuse_port
#endif
       ) 
    {   
```